patch/series

```diff
@@ -70,6 +70,7 @@ Support-for-fullcone-nat.patch
 # Tbridge-per-port-multicast-broadcast-flood-flags.patch
 #
 
+0001-prevent-bridge-link-show-failure.patch
```

Comment:
Also, please use the name created by `git format-patch`, so that it contains the whole subject line.

Reply:
Updated